README.md

```diff
@@ -1 +1,37 @@
-# apprenticeship-base-repo
+# InstaClone
+*This is a webpage where we can add our posts.*
+*This project was bootstrapped with Create React App.*
+
+## **prerequisite**
+Visual Studio Code (source-code editor) <br />
+Node.js <br />
+Node.js <br />
+HTML    <br />
+CSS     <br />
+Git     <br />
+
+##Scripts
+In this directory You can run:
+
+### 'npm install'
+This will install Node.js 
```

Comment:
It installs the packages required for the nodejs application. NodeJS should be there installed on our machines to use the command npm.